internal/cmd/server/server.go

```diff
@@ -14,41 +15,66 @@ func NewCommand(s state.State) *cobra.Command {
 		TraverseChildren:      true,
 		DisableFlagsInUseLine: true,
 	}
-	cmd.AddCommand(
+
+	util.AddGroup(cmd, "general", "General",
 		ListCmd.CobraCommand(s),
 		DescribeCmd.CobraCommand(s),
 		CreateCmd.CobraCommand(s),
 		DeleteCmd.CobraCommand(s),
-		RebootCmd.CobraCommand(s),
-		PoweronCmd.CobraCommand(s),
-		PoweroffCmd.CobraCommand(s),
-		ResetCmd.CobraCommand(s),
-		ShutdownCmd.CobraCommand(s),
-		CreateImageCmd.CobraCommand(s),
-		ResetPasswordCmd.CobraCommand(s),
-		EnableRescueCmd.CobraCommand(s),
-		DisableRescueCmd.CobraCommand(s),
-		AttachISOCmd.CobraCommand(s),
-		DetachISOCmd.CobraCommand(s),
 		UpdateCmd.CobraCommand(s),
+		CreateImageCmd.CobraCommand(s),
 		ChangeTypeCmd.CobraCommand(s),
 		RebuildCmd.CobraCommand(s),
-		EnableBackupCmd.CobraCommand(s),
-		DisableBackupCmd.CobraCommand(s),
-		EnableProtectionCmd.CobraCommand(s),
-		DisableProtectionCmd.CobraCommand(s),
-		SSHCmd.CobraCommand(s),
 		LabelCmds.AddCobraCommand(s),
 		LabelCmds.RemoveCobraCommand(s),
-		SetRDNSCmd.CobraCommand(s),
+	)
+
+	util.AddGroup(cmd, "protection", "Protection",
+		EnableProtectionCmd.CobraCommand(s),
+		DisableProtectionCmd.CobraCommand(s),
+	)
+
+	util.AddGroup(cmd, "rescue", "Rescue",
+		EnableRescueCmd.CobraCommand(s),
+		DisableRescueCmd.CobraCommand(s),
+	)
+
+	util.AddGroup(cmd, "power", "Power/Reboot",
+		PoweronCmd.CobraCommand(s),
+		PoweroffCmd.CobraCommand(s),
+		RebootCmd.CobraCommand(s),
+		ShutdownCmd.CobraCommand(s),
+		ResetCmd.CobraCommand(s),
+	)
+
+	util.AddGroup(cmd, "network", "Networks",
 		AttachToNetworkCmd.CobraCommand(s),
 		DetachFromNetworkCmd.CobraCommand(s),
 		ChangeAliasIPsCmd.CobraCommand(s),
+		SetRDNSCmd.CobraCommand(s),
```

Comment:
In other commands, we put the SetRDNS command in the misc sections, do you think we should always put the setRDNS command in a network Section ?

Reply:
Good question. I think we can move it to misc here too, since it's not really connected to networks as a resource anyway.